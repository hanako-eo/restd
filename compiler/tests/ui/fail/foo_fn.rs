#[compiler::item("foo_fn")]
fn foo() {}

fn main() {}
