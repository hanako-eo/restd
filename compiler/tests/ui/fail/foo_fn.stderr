error: `foo_fn` item expected to have 1 arguments but the definition has 0 arguments
        please force the function to have 1 arguments
 --> tests/ui/fail/foo_fn.rs:2:1
  |
2 | fn foo() {}
  | ^^^^^^^^
